docs/en/ingest-management/elastic-agent/scaling-on-kubernetes.asciidoc

```diff
@@ -145,9 +145,21 @@ Policy configuration of {k8s} package can heavily affect the amount of metrics c
 - Disable events dataset
 - Disable {k8s} control plane datasets in Cloud managed {k8s} instances (see more info ** <<running-on-gke-managed-by-fleet>>, <<running-on-eks-managed-by-fleet>>, <<running-on-aks-managed-by-fleet>> pages)
 
+
+[discrete]
+=== Dashboards and Visualisations 
+
+https://github.com/elastic/integrations/blob/main/docs/dashboard_guidelines.md[Dashboard Guidelines] document can provide guidance on how to implement your dashboards and is constantly updated also to track needs of observability at scale.
+
 User experience regarding Dashboard responses, is also affected from the size of data being requested. As dashboards can contain multiple visualisations, the general consideration is to split visualisations and group them according to the frequency of access. The less number of visualisations tends to improve user experience.
 
-Additionally, https://github.com/elastic/integrations/blob/main/docs/dashboard_guidelines.md[Dashboard Guidelines] is constantly updated also to track needs of observability at scale.
+[discrete]
+=== Disabling indexing host.ip and host.mac fields 
+
+A new environemntal variable `NETINFO: false` has been introduced to globally disable the indexing of fields `host.ip` and `host.mac` in your {k8s} integration. For more information see <<agent-environment-variables>>.
```

Comment:
Does it disable indexing all `host.ip` fields or just reduces the number?

Reply:
Disables them